source/documentation/other-topics/rds-external-access.html.md.erb

```diff
@@ -57,10 +57,9 @@ Use port `1433` for MS-SQL.
 
 > This creates a pod in your namespace, but without a deployment to manage it.
 >
-> Manually-created pods (i.e. those which are not part of a [deployment]) will be deleted
-by regular cluster maintainance operation - [node-recycling] which runs every weekday between 8 am and 10 am.
-If you need a port-forward pod to persist longer than
-this, please make it part of a [deployment].
+> Manually-created pods (i.e. those which are not part of a [deployment]) will always be _eventually_ deleted
+by regular cluster maintainance operation - [node-recycling], which runs every weekday between 8 am and 10 am.
```

Comment:
```suggestion
by regular cluster maintainance operation - [node-recycling], which runs every weekday between 8 am and 10 am. This recycles one node at a time, so it can take upwards of 60 days to delete.
```